src/System.IO.FileSystem/tests/File/Delete.cs

```diff
@@ -122,6 +124,30 @@ public void Unix_NonExistentPath_Nop()
         }
 
         [Fact]
+        [OuterLoop("Needs sudo access")]
+        [PlatformSpecific(TestPlatforms.Linux)]
+        [Trait(XunitConstants.Category, XunitConstants.RequiresElevation)]
+        public void Unix_NonExistentPath_ReadOnlyVolume()
+        {
+            ReadOnly_FileSystemHelper(readOnlyDirectory =>
+            {
+                Delete(Path.Combine(readOnlyDirectory, "DoesNotExist"));
+            });
+        }
+
+        [Fact]
+        [OuterLoop("Needs sudo access")]
```

Comment:
Why outerloop if its already got the elevated trait so it would not normally run locally